WorkflowRxSwift.podspec

```diff
@@ -19,7 +19,9 @@ Pod::Spec.new do |s|
   s.source_files = 'WorkflowRxSwift/Sources/**/*.swift'
 
   s.dependency 'Workflow', "#{s.version}"
-  s.dependency 'RxSwift', '~> 5.1.1'
+  s.dependency 'RxSwift', '>= 4.4.0'
+  s.dependency 'RxCocoa', '>= 4.4.0'
+  s.dependency 'RxAtomic', '>= 4.4.0'
```

Comment:
Sorry, one more question. Do we actually use RxCocoa and RxAtomic directly?

Reply:
Nope, but it's needed for `RxSwift`. `WorkflowRxSwift` failed to compile without these.

Reply:
That’s weird. So RxSwift depends them but doesn’t declare them as dependencies? FWIW I couldn’t repro that problem locally removing these and doing `pod gen`

Reply:
`pod gen` is probably using `5.1.1`. If you could pin it to `4.4.0`, the problem repros (I think).

Reply:
`RxSwift` depends upon `RxAtomic`. It's surprising if it's fails to compile without `RxCocoa` if y'all aren't using its objects since it lives above `RxSwift`

Reply:
@sebastianv1: You're right, didn't need `RxCocoa`. Removing.

Reply:
Thanks! Kinda silly about the RxAtomic thing, but we can’t go back in time and fix their bug, so. Onward to RxSwift 5.x!

Reply:
Bug? I think it was more of a separation for atomic primitives that could be shared without consuming `RxSwift`.

Reply:
The bug being `RxSwift` doesn’t declare a dependency on a module that must be present for it to compile.

Reply:
Looks like it does? https://github.com/ReactiveX/RxSwift/blob/4.4.0/RxSwift.podspec#L38
Definitely a little hidden at the bottom though.

Reply:
Looks like it was this issue: https://github.com/ReactiveX/RxSwift/issues/1878